Makefile

```diff
@@ -6,6 +6,57 @@ check_version:
 check_tag:
 	./scripts/check_tag
 
+.PHONY: nano_testnet_build
+nano_testnet_build:
+	FULLNODE_HOST=node1.nano-testnet.hathor.network; \
+	export REACT_APP_BASE_URL=https://$$FULLNODE_HOST/v1a/; \
+	export REACT_APP_WS_URL=wss://$$FULLNODE_HOST/v1a/ws/; \
+	export REACT_APP_EXPLORER_SERVICE_BASE_URL=https://explorer-service.nano-testnet.hathor.network/; \
+	export REACT_APP_TIMESERIES_DASHBOARD_ID=59683ac0-237a-11ef-8f75-578bca86e218; \
+	export REACT_APP_NETWORK=nano-testnet-alpha; \
+	npm run build
```

Comment:
suggestion(if-minor): maybe move this to a build script.

The idea is to have all of these parameters pre-configured on the script so we don't repeat this environment configuration on every build command and the command can be a simple `make build site=testnet` and the build script will get the site parameter and use the correct configuration.

Adding others will be simpler since we would just need to add the new site config and build with the `site="new site name"`

What do you think?

Reply:
Done in https://github.com/HathorNetwork/hathor-explorer/pull/274/commits/086c5605c739373d9f323962652e37862b539b47